.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure if this should be put on VCS or not, we'll have to check with the others

Reply:
I don't think it should : it's a file related to the IDE you are using.